configs/metadata.yaml

```diff
@@ -39,51 +39,86 @@ s2:
       swir16: 450
       swir22: 480
     wavelength:
-      blue: 0.4905
-      green: 0.5605
+      blue: 0.49
```

Comment:
Only one nitpick change for Sentinel-2, referencing https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-2-msi/resolutions/spectral. Other bands look ok.

![image](https://github.com/Clay-foundation/model/assets/23487320/d3c1fa88-a16d-4361-93c0-507501b3d4c2)

```suggestion
      blue: 0.493
```